bundles/org.openhab.binding.magentatv/README.md

```diff
@@ -0,0 +1,222 @@
+# MagentaTV Binding (aka TelekomTV aka EntertainTV)
+
+---
+
+## Release: 2.5-pre5
```

Comment:
Please remove this section from the official docs

Reply:
done